### PR TITLE
Actions and badges

### PR DESCRIPTION
**Pull request by bandophahita**
Adding github actions & badges

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+jobs:
+  run-tests:
+    name: Python ${{ matrix.python-version }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 9
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Comment (perrygoy) [inline .github/workflows/tests.yml:19]: Should we add 3.11, now that it's out?

File: tox.ini [hunk 2]
```diff
@@ -11,6 +11,13 @@ envlist =
     py310
 isolated_build = True
 
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
```
Comment (perrygoy) [inline tox.ini:19]: Same question here about 3.11.

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```
Comment (perrygoy) [inline .github/workflows/lint.yml:17]: Same question here about 3.11.

**Review by perrygoy — APPROVED** [review]
Thanks again for adding this to the project!

File: tox.ini (hunk 2 above)
Comment (bandophahita) [reply tox.ini:19]: Support for a new python version felt like it was deserving of it's own PR since there may be other code needed.

Comment (perrygoy) [reply tox.ini:19]: That is definitely the smarter way. :P

**Pull request merged**